crates/torii/migrations/20230316154230_setup.sql

```diff
@@ -75,6 +75,19 @@ CREATE INDEX idx_entities_keys ON entities (keys);
 
 CREATE INDEX idx_entities_event_id ON entities (event_id);
 
+CREATE TABLE event_messages (
```

Comment:
Since migrations are incremental, this should be part of the new `20240314182410_event_model.sql`